### ISSUE TITLE
Combo buttons not working

### ISSUE THREAD

**Issue by bennydiamond** [XorTroll/emuiibo#271]
Hi,

I'm trying to use the combo buttons input to quickly control emuiibo (R3 + dpab button) but none of the 4 combos work, in TotK at least.

Emuiibo overlay states
Emulation is active
Game is intercepted
Target Amiibo is active

Whenever I try to press R3 + dpad left to cycle Amiibo, nothing is happening. The same is true for any other combo command.

I tried browsing other GH issues but all I foudn was some very old (and closed) issue of some people having issues with the combo buttons as well. There is mention of a "settings.json" file on MicroSD. Couldn't find one on mine and I suspect this is no longer relevant.

Known issue?
Do I need to explicitly configure Combo buttons to be ON somewhere?

Running: 

- TotK 1.2.1
- Hekate 6.1.0 
- Atmosphere 1.6.2
- emuiibo 1.1.1
- Basically all up to date concerning Tesla menu a ovlloader. I modded my switch this week so it's all fresh stuff on it.
- HOS 17.0.1

Thanks
